configurations/s02_pipeline_configuration.py

```diff
@@ -371,7 +383,29 @@
                       )
                     ),
                 ]
-            )
+            ),
+            AnalysisDatasetConfiguration(
+                engagement_db_datasets=["disability", "disabled"],
```

Comment:
How did we end up with 2 names for this?

Reply:
It's because we have:
1. coda dataset configuration with coda dataset id `WUSC-KEEP-II_kakuma_disabled`  and engagement db dataset set to `disabled`.
2. and this analysis config with engagement db set to `disability`